### DISCUSSION TITLE
Delegate to multiple validators in one transaction.

### DISCUSSION THREAD

**Discussion by collincusce** [avalanche-foundation/ACPs#51] [Brainstorming]
Context and scope
Delegate to multiple validators in one transaction.

A single validator can fail to meet uptime goals. Spreading delegation to multiple would be more ideal. There should be a web wallet interface for this so you can select the validators based on past performance and time remaining. The transaction should just be a list of validators and amount to delegate to them.

**Comment by StephenButtolph** [comment]
I wonder if it makes sense to delegate to _multiple_ validators - or rather "delegate" to _all_ the validators? Having some notion of a P-chain treasury bond or something rather than the existing delegation mechanism has always fascinated me.

**Reply by collincusce** [reply]
I had originally considered that but the reason I didn't go that route is because with the current system you can choose a percentage value to take from the delegation rewards and doing it across the entire network would require incentives which might conflict with that system.
